acquaman_suite.pro

```diff
@@ -29,7 +29,7 @@ SUBDIRS += \
 	SXRMBAddOnsCoordinator.pro \
 	VESPERSAcquaman.pro \
 	VESPERSBendingMirrors.pro \
-	VESPERSDatabaseDuplicateEntryPatch.pro \
+#	VESPERSDatabaseDuplicateEntryPatch.pro \
```

Comment:
Quick question--would VESPERS be negatively affected by removing this .pro file?